doc/sphinx/user/extending/testing/running-tests.md

```diff
@@ -105,7 +105,7 @@ This command executes the shell script `cmake/compile_and_update_tests.sh`
 *inside* the docker container that contains the official
 ASPECT test system. Note that by mounting your
 ASPECT folder into the container you are actually
-updating the reference test results on the host system (i.e. your computer).
+updating the reference test results on the host system (i.e., your computer).
```

Comment:
I had to look this up, because I would not do this myself, but you are right in American English there is a comma (in British English there is not). https://www.dailywritingtips.com/comma-after-i-e-and-e-g/